keyboards/crkbd/keymaps/default/keymap.c

```diff
@@ -19,52 +19,59 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include QMK_KEYBOARD_H
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-    [0] = LAYOUT_split_3x6_3(
-  //,-----------------------------------------------------.                    ,-----------------------------------------------------.
-       KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                         KC_Y,    KC_U,    KC_I,    KC_O,   KC_P,  KC_BSPC,
-  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      KC_LCTL,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                         KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN, KC_QUOT,
-  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
-      KC_LSFT,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                         KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH,  KC_ESC,
-  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
-                                          KC_LGUI,   MO(1),  KC_SPC,     KC_ENT,   MO(2), KC_RALT
-                                      //`--------------------------'  `--------------------------'
+    [0] = LAYOUT_split_3x6_3_ex2(
```

Comment:
You cannot make this change as it makes the "generic" keymap fail on all the other revisions as they do not support `LAYOUT_split_3x6_3_ex2`. Keymaps would have to be moved to be per revision, or a `LAYOUT_split_3x6_3` implementation added to rev2 so this keymap compiles without changes on all revisions.

Reply:
Add `LAYOUT_split_3x6_3` to rev2
https://github.com/qmk/qmk_firmware/pull/22676/commits/ac38e912877e1b78fe7a712412330c3d3a6df71e